<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-LOG2SSBM" name="omg.SecondTest" tests="6" failures="0" timestamp="22 Jul 2020 13:11:06 GMT" time="84.436" errors="1">
  <testcase name="fgtry" time="11.531" classname="omg.SecondTest"/>
  <testcase name="boil" time="25.858" classname="omg.SecondTest"/>
  <testcase name="fgrt" time="20.439" classname="omg.SecondTest">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method omg.SecondTest.fgrt() didn&#039;t finish within the time-out 3000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method omg.SecondTest.fgrt() didn't finish within the time-out 3000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNoExecutor(MethodInvocationHelper.java:283)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:265)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:589)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- fgrt -->
  <testcase name="huio" time="8.955" classname="omg.SecondTest"/>
  <testcase name="frt" time="8.903" classname="omg.SecondTest"/>
  <testcase name="ouil" time="8.750" classname="omg.SecondTest"/>
</testsuite> <!-- omg.SecondTest -->
